_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -97,10 +97,10 @@ This button has [visible][] text that is contained within the [accessible name][
 
 #### Passed Example 5
 
-This button has [visible][] text that does not need to be contained within the [accessible name][], because the "x" text node is [non-text content][].
+This button has [visible][] text that does not need to be contained within the [accessible name][], because the "x" text node is [non-text content][]. Note: this would need to meet SC 1.1.1 Non text content.
```

Comment:
Should we be listing SC 1.1.1 as a secondary requirement? Feels like we are leaning no, but figure we should consider it if its possibly related.

Reply:
I think SC 1.1.1 would be a secondary requirement, but I'd like to leave that for a separate PR.